all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -12,52 +12,63 @@
 # the License.
 #
 
-# This template uses Jaeger with in-memory storage with limited functionality
-# Do not use this in production environment!
-#
-# oc create -f jaeger-all-in-one-template.yml
-# oc process -f jaeger-all-in-one-template.yml | oc create -f -
-# oc delete all,template -l jaeger-infra
 parameters:
-- description: The name of the Jaeger
-  displayName: Jaeger service name
-  name: JAEGER_NAME
+- description: The name of the Jaeger Service.
+  displayName: Jaeger Service Name
+  name: JAEGER_SERVICE_NAME
+  required: true
+  value: jaeger
+- description: The namespace where to install the Jaeger instance
+  displayName: Namespace
+  name: NAMESPACE
```

Comment:
Could we make this optional? By default install it in the current namespace? It should be possible with prod template too, then the instructions would be more specific and clear from beginning that it will use jaeger-infra namespace.